### DISCUSSION TITLE
Add form fields to a field within twig?

### DISCUSSION THREAD

**Discussion by Larsundso** [solspace/craft-freeform#932] [Questions & Answers] [answered]
Pseudocode:
```
{{ craft.freeform.form("composerForm", {
	additionalFields: [
		{
		 	handle: "test-handle",
			label: "test-table",
			type: "checkbox",
		}
	]
}).render() }}
```

Can I use something like above to add fields to a form that haven't been added to the form using the form builder?
In my use-case I add form fields from provided URL parameters, so I can't know beforehand which fields to add to the form builder.

**Comment by kjmartens** [comment, marked as answer]
Hi @Larsundso,

Unfortunately something like this is not possible. It might be better to just include a handful of extra hidden fields in the form builder to get populated by the URL params. 🙂
